src/test/rpc/AMMInfo_test.cpp

```diff
@@ -316,13 +316,45 @@ class AMMInfo_test : public jtx::AMMTestBase
         });
     }
 
+    void
+    testInvalidAmmField()
+    {
+        using namespace jtx;
+        testcase("Invalid amm field");
+
```

Comment:
Consider refactoring to simplify and for consistency with other tests:

        testAMM([&](AMM& amm, Env&) {
            auto const resp = amm.ammRpcInfo(
                std::nullopt,
                jss::validated.c_str(),
                std::nullopt,
                std::nullopt,
                gw);
            BEAST_EXPECT(resp.isMember("error") && resp["error"] == "actNotFound");
        });